composer.json

```diff
@@ -31,7 +31,7 @@
 		"ext-libxml": "*",
 		"ext-mbstring": "*",
 		"ext-simplexml": "*",
-		"league/flysystem": "^1.0",
+		"league/flysystem": "^1.0|^2.0|^3.0",
```

Comment:
Probably last request, as the rest looks absolutely brilliant.
Now that `League\MimeTypeDetection` classes are being used explicitly in QTI-SDK's codebase, could you also include `"league/mime-type-detection": "^1.0"` here?